Makefile

```diff
@@ -5,3 +5,4 @@ clean  :; dapp clean
 test   :; ./test-dssspell.sh $(match)
 deploy :; make && dapp create DssSpell | xargs ./verify.py DssSpell
 flatten :; hevm flatten --source-file "src/Goerli-DssSpell.sol" > out/flat.sol
+archive-spell :; ./archive-spell.sh $(name)
```

Comment:
should we keep the `dssspell` naming convention in the script file?

Reply:
yeah i'll update the file name, good call

Reply:
Done in my PR after merging this.